obal/data/roles/build_srpm/tasks/main.yaml

```diff
@@ -1,10 +1,41 @@
 ---
+- name: 'Set build_srpm_tito_releaser_args'
+  set_fact:
+    build_srpm_tito_releaser_args: "{{ build_package_tito_releaser_args | default([]) }}"
```

Comment:
you define `build_srpm_tito_releaser_args: []` in `defaults`, which makes me think the user is allowed to set `build_srpm_tito_releaser_args: ['--whatever']`. but this `set_fact` will override it?

Reply:
Hrm, yeah that does seem a bit goofy. Let me clean this up.

Reply:
defaults to `build_package_tito_releaser_args` but overrides with passed in `--dist` and/or `--scl` flags, or `build_srpm_tito_releaser_args` if you like extra vars.